CHANGELOG.md

```diff
@@ -1,21 +1,30 @@
+# 0.5.7
+
+* Added `Socket::(set_)passcred`
+  (https://github.com/rust-lang/socket2/pull/506).
+* Added `RecvFlags::is_confirm` and `RecvFlags::is_dontroute`
+  (https://github.com/rust-lang/socket2/pull/499).
+* Added `MsgHdrMut::control_len`
+  (https://github.com/rust-lang/socket2/pull/505).
+
 # 0.5.6
 
-* Add `Socket::(set_)multicast_all_v{4,6}`
+* Added `Socket::(set_)multicast_all_v{4,6}`
   (https://github.com/rust-lang/socket2/pull/485 and
```

Comment:
This seems to change a convention for the changelog. You don't want a separate PR for that?

Reply:
It makes the convention consistent, we use both `Add` and `Added` before. I put it in a separate commit, which I'm not squashing, so I think it's fine.